app/src/androidTest/java/ch/sdp/vibester/activity/SearchUserActivityTest.kt

```diff
@@ -98,6 +185,32 @@ class SearchUserActivityTest {
 
         override fun getDescription() = "Click on a child view with specified id."
 
-        override fun perform(uiController: UiController, view: View) = click().perform(uiController, view.findViewById<View>(viewId))
+        override fun perform(uiController: UiController, view: View) = click().perform(uiController, view.findViewById(viewId))
+    }
+
+    /**
+     * Custom functions to match the item views inside Recycle View
+     */
+    fun checkRecyclerSubViews( recyclerViewId: Int, position: Int, itemMatcher: Matcher<View?>, subViewId: Int) {
+        onView(withId(recyclerViewId)).perform(
+            RecyclerViewActions.scrollToPosition<RecyclerView.ViewHolder>(
+                position
+            )
+        )
+            .check(matches(atPositionOnView(position, itemMatcher, subViewId)))
```

Comment:
maybe move this line above for readability

Reply:
done!